network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/youtube_regrets_2022.html

```diff
@@ -0,0 +1,26 @@
+{% extends "pages/base.html" %}
+{% load wagtailcore_tags %}
+
+{% block body_id %}youtube-regrets-2022{% endblock %}
+{% block bodyclass %}youtube-regrets-2022{% endblock %}
+
+{% block extended_head %}
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Changa:wght@600;700&display=swap" rel="stylesheet">
+{% endblock %}
+
+{# Empty header; Do not add <header role="banner"> to the page #}
+{% block header_wrapped %}{% endblock %}
+
+{% block content_wrapped %}
+  <main role="main">
```

Comment:
Ah I never noticed this since I just copied the last year template but, honestly I don't think the role="main" helps at all cause the role is already main from the element being semantically `<main>` hahah nice one

Reply:
I've removed this now on last year and this year's template